packages/patternfly-4/react-core/src/components/DataList/DataListToggle.js

```diff
@@ -8,13 +8,15 @@ import { Button } from '../Button';
 const DataListToggle = ({
   className,
   isExpanded,
+  'aria-controls': ariaControls,
   'aria-label': ariaLabel,
   'aria-labelledby': ariaLabelledBy,
+  rowid,
```

Comment:
I'm seeing `rowid` display as an attribute in almost every child html element of `li.

For example, in this snippet, you can see `rowid` which shouldn't be there. However the `id` with the same value on "Primary content" should be there.
```
<div class="pf-c-data-list__cell" rowid="ex-item1">
  <div id="ex-item1">Primary content</div>
  <a href="#">link</a>
</div>
```

Reply:
Yeah, that's because you have to pass the id down from the parent to all the children. Let me see if I can restructure the div to show everything except that prop.